app/src/pages/authPages/Address/index.js

```diff
@@ -66,6 +67,17 @@ export default function Address({ route, navigation }) {
           setCity(localidade);
           setComplement( logradouro + " / " + bairro );
         } else {
+          // alert("CEP não encontrado!");
```

Comment:
@LhTaira,Testei aqui e ta tudo certo, só retirar esse comentário aqui se ele não for necessário e pode mergear. o/